src/main/webapp/singleDrug.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
If singleDrug.html is going to live under /drugs/ then you can remove this copy.